dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1126,24 +1165,19 @@ sources:
                   meta:
                     description: >
                       user20 (No. of Commercial Units) should not be null
-          - name: user24
-            description: '{{ doc("shared_column_card_proration_rate") }}'
-            tests:
-              - not_null:
-                  name: iasworld_dweldat_card_proration_rate_not_null
+              - accepted_values:
+                  name: iasworld_dweldat_char_ncu_between_0_and_5
```

Comment:
This test is not new, but rather has been moved from an `expression_is_true` test that was defined on the model and translated into an `accepted_values` test that is defined on the column. I'm preferring to move tests from the model level to the column level where possible, since it allows the DAG to determine and display which tests are defined on a column in the docs; happy to discuss more if you disagree with that preference.

Reply:
That makes sense to me!